packages/checkout/widgets-lib/src/widgets/wallet/views/WalletBalances.tsx

```diff
@@ -264,6 +264,7 @@ export function WalletBalances({
           onCloseDrawer={() => setShowNotEnoughGasDrawer(false)}
           walletAddress={walletAddress}
           showAdjustAmount={false}
+          tokenSymbol={ETH_TOKEN_SYMBOL}
```

Comment:
Wallet widget shows a not enough gas drawer when `Move` is selected but the user has no ETH - so just adding the token symbol here